benchmarks/src/main/scala/scalaz/zio/IOBenchmarks.scala

```diff
@@ -2,13 +2,20 @@ package scalaz.zio
 
 import cats._
 import cats.effect.{ Fiber => CFiber }
+
 import scala.concurrent.ExecutionContext
 import cats.effect.{ ContextShift, IO => CIO }
 import monix.eval.{ Task => MTask }
 import scalaz.zio.internal._
 
 object IOBenchmarks extends DefaultRuntime {
-  override val Platform: Platform = PlatformLive.makeDefault(Int.MaxValue)
+  override val Platform: Platform = PlatformLive
+    .makeDefault(Int.MaxValue)
+    .withTracing(Tracing.disabled)
```

Comment:
Would be good to add `withTracing` to `Runtime`, too, which has other similar helpers for every method on `Platform`. 👍